### ISSUE TITLE
Render distance abuse

### ISSUE THREAD

**Issue by Mithallas** [SmartlyDressedGames/Unturned-3.x-Community#1229]
Players can see where other players are hiding because the render distance setting only affects objects, not other players. It is quite difficult for admins to discern between players abusing this and players using cheats, due to very similar playstyle.

One suggestion would be to add a server config for minimum render distance.  The default can still be the lowest settings of course, I doubt most server admins will ever touch it.
Thanks for reading :)

![Screenshot_2](https://user-images.githubusercontent.com/35383961/69003978-364a8400-08d9-11ea-9126-c2aeec6651b5.png)

**Comment by SDGNelson** [comment]
I think at one point players were somewhat affected by render distance, but it was annoying having them pop in after getting close. Preferably clients have full control over graphics, though this is a problem... will think about it.

**Comment by SDGNelson** [comment]
It's been discussed a few more times since then. I think we might need to raise the minimum LOD bias, since it looks bad and the default is really low.

**Comment by Mithallas** [comment]
while we're on the issue of rendering, for a while I have noticed that there are always large frames drops when rendering large bases, even with higher end gfx cards.  I don't know much about rendering but I've been told minecraft had success with something called pre-rendering? I can't speak for all hosters, but I would be more than happy if server CPU/RAM usage went up a bit to give players a better experience. I did search issues history and found another who described something similar, although it was closed without much detail. 

https://github.com/SmartlyDressedGames/Unturned-3.x-Community/issues/164#issue-286549558
